### PR TITLE
Feature/ines thank you heart

### PR DESCRIPTION
**Pull request by inescoelho**
Add a page with a collage of photos to thank everyone involved with RGSoC 2017!

### CODE REVIEW

File: thank-you-2017/index.md
Comment (alicetragedy) [inline thank-you-2017/index.md:2]: is there a reason for using a completely blank layout as opposed to having this embedded into the website as a regular page (for example, like the sponsor page or the campaign page)? 
I think that further down the line, when we make changes to the website styling and structure, it will be easier to have this page use one of the existing layouts and show the menu bar, as opposed to being a completely separate, standalone thing.

File: thank-you-2017/heart.js
Comment (alicetragedy) [inline thank-you-2017/heart.js]: it would be great if you could use the existing structure we have for libraries, css and javascript. I know it's a lot of cleaning up work, but I think it would really be best to have everything in the respective folders (`assets/javascripts`, `assets/stylesheets`) and to move the images as well to the `img` folder. Could you do that?
Ideally also sticking to the convention of using scss :)

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Hey @inescoelho, thanks so much for all your work with the thank you heart! It looks awesome and I can't wait to have it up and share it :)
Before we merge this, I was wondering if it would be possible to do two things:
- embed the page as a “regular” page, so that it would also have the menu bar as every other page does.
- clean up the structure a bit as commented — basically moving all the assets where they belong in our current structure, rather than making this a separate folder that all files are dumped into.

I know it's quite a bit of work, and if you can't do it right now, that's alright (we can create an issue to clean up the code instead) but I just wanted to point it out because I think it will make it easier to maintain :) Let me know if you have any questions or if I can help!

**Comment by inescoelho** [comment]
@alicetragedy I've made it! 
I just couldn't figure out how to transform my css file in scss, the browser rejects my scss file. So, I've committed my css file instead.

**Review by alicetragedy — APPROVED** [review]
Looks good, thank you for the changes!
I think that's good enough, as I know it's time-sensitive — we can make some improvements later when there's more time. 👍

**Comment by alicetragedy** [comment]
@inescoelho what do you think — should we merge this already? Then we can make sure everything works nicely and that the link for #445 is correct.

**Comment by inescoelho** [comment]
Yes, go ahead!

**Comment by alicetragedy** [comment]
dancing-panda-small

**Pull request merged**
